bson/src/main/org/bson/Document.java

```diff
@@ -176,6 +192,18 @@ public Long getLong(final Object key) {
     }
 
     /**
+     * Gets the value of the given key as a Long.
+     *
+     * @param key          the key
+     * @param defaultValue what to return if the value is null
+     * @return the value as an Long, which may be null
+     * @throws java.lang.ClassCastException if the value is not an Long
+     */
+    public Long getLong(final Object key, final Long defaultValue) {
```

Comment:
I'd prefer a signature similar to getInteger just above:  

     long getLong(final Object key, final long defaultValue)

If the user wants a non-null default value, there's no reason to use the boxed type